incubator-tools/amount_rectification_from_words_to_numbers/amount_rectification_from_words_to_numbers.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.    def words_nearest_match(text: str) -&gt; str:
</pre><p>Surely there's an existing library for this? Can it be used instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Tried existing libraries but didnt work , so customized and created a new one to work for our needs

Reply:
Was it possible to use fuzzywuzzy for fuzzy string matching?

https://pypi.org/project/fuzzywuzzy/